src/test/compile-fail/issue-52060.rs

```diff
@@ -0,0 +1,6 @@
+// Regression test for https://github.com/rust-lang/rust/issues/52060
```

Comment:
Great! Can you move this to `src/test/ui/issues`, though? We're not really using `compile-fail` anymore (though we haven't fully killed it yet)